charts/reform-scan-notification-service/values.preview.template.yaml

```
@@ -39,10 +39,11 @@ java:
     postgresqlPassword: db_password
     postgresqlDatabase: db_database
 servicebus:
-  resourceGroup: reform-scan-aks
+  resourceGroup: reform-scan-aso-preview-rg
   teamName: "BSP"
   location: uksouth
   serviceplan: basic
+  sbNamespace: reform-scan-sb-preview
```

Hey, sorry there's an annoying bug in kustomize at the moment where it's not handling other namespace resources correctly - so the resource is called reform-scan on the cluster
```suggestion
  sbNamespace: reform-scan
```